examples/threadExample/src/main/res/layout/fragment_async_query.xml

```diff
@@ -0,0 +1,43 @@
+<!--
```

Comment:
This header looks different from the others, using `~` instead of `*`. Any reason for this?